pinecone/config.py

```diff
@@ -242,18 +242,71 @@ def init(
     project_name: str = None,
     log_level: str = None,
     openapi_config: OpenApiConfiguration = None,
-    config: str = "~/.pinecone",
+    config: str = "./.pinecone",
     **kwargs
 ):
-    """Initializes the Pinecone client.
-
-    :param api_key: Required if not set in config file or by environment variable ``PINECONE_API_KEY``.
-    :param host: Optional. Controller host.
-    :param environment: Optional. Deployment environment.
-    :param project_name: Optional. Pinecone project name. Overrides the value that is otherwise looked up and used from the Pinecone backend.
-    :param openapi_config: Optional. Set OpenAPI client configuration.
-    :param config: Optional. An INI configuration file.
-    :param log_level: Deprecated since v2.0.2 [Will be removed in v3.0.0]; use the standard logging module to manage logger "pinecone" instead.
+    """Initializes configuration for the Pinecone client.
+
+    The `pinecone` module is the main entrypoint to this sdk. You will use instances of it to create and manage indexes as well as 
+    perform data operations on those indexes after they are created.
+
+    **Initializing the client**
+
+    There are two pieces of configuration required to use the Pinecone client: an API key and environment value. These values can
+    be passed using environment variables, an INI configuration file, or explicitly as arguments to the ``init`` function. Find
+    your configuration values in the console dashboard at [https://app.pinecone.io](https://app.pinecone.io).
+    
+    **Using environment variables**
+
+    The environment variables used to configure the client are the following:
+    
+    ```python
+    export PINECONE_API_KEY="your_api_key"
+    export PINECONE_ENVIRONMENT="your_environment"
+    export PINECONE_PROJECT_NAME="your_project_name"
+    export PINECONE_CONTROLLER_HOST="your_controller_host"
+    ```
+
+    **Using an INI configuration file**
```

Comment:
Slipped a section in here for setting up an INI file. I don't think this was documented at all previously.